external/mbedtls/ecdh.c

```diff
@@ -222,13 +213,12 @@ int mbedtls_ecdh_get_params( mbedtls_ecdh_context *ctx, const mbedtls_ecp_keypai
         ( ret = mbedtls_mpi_copy( &ctx->d, &key->d ) ) != 0 )
         return( ret );
 
-#if defined(CONFIG_TLS_WITH_SSS)
+#if !defined(MBEDTLS_ECDH_GEN_PUBLIC_ALT) && !defined(MBEDTLS_ECDH_COMPUTE_SHARED_ALT)
 	ctx->grp.key_index = key->key_index;
```

Comment:
Could you please check below errors. 

CC:  ecdh.c
ecdh.c: In function 'mbedtls_ecdh_get_params':
ecdh.c:217:10: error: 'mbedtls_ecp_group {aka struct <anonymous>}' has no member named 'key_index'
  ctx->grp.key_index = key->key_index;
          ^
ecdh.c:217:26: error: 'mbedtls_ecp_keypair {aka const struct <anonymous>}' has no member named 'key_index'
  ctx->grp.key_index = key->key_index;


After than this, I will try to run on device.